### ISSUE TITLE
Using this in a project

### ISSUE THREAD

**Issue by WACriminalG2** [Levertion/minecraft-json-schemas#14]
Wasn't sure of the best way to contact you all. Just wanted to let you know I'm working on integrating modified versions of these schemas into a VScode extension I'm publishing for modpack devs: https://marketplace.visualstudio.com/items?itemName=PackDevUtils.packdevutils . Gave you credit in the changelog and left your URLs intact in the schema $ids.

**Comment by Levertion** [comment]
Yeah, that's fine. The best way to contact me is probably via the MCC discord - this GitHub account is mostly dormant.
